ui/src/app/shared/job-stream.ts

```diff
@@ -36,7 +36,7 @@ export class JobStream extends BehaviorSubject<JobListView> {
         return prevResp;
       }
       let pageSize = Math.max(
-        JobStream.minBackendPageSize, this.value.results.length - atLeast);
+        JobStream.minBackendPageSize, Math.abs(this.value.results.length - atLeast));
```

Comment:
Did you mean to leave this in? I can't rationalize why you would need this change.

Reply:
This is needed for the case where there needs to be a request for more jobs and `JobStream.minBackendPageSize` is smaller than `pageSize` (defined in `JobListComponent`).

Reply:
OK, there is a bug here which is that the order was incorrect, should be `atLeast - ...results.length`

Reply:
Please use the above fix instead.

Reply:
Are there really no circumstances where `this.value.results.length` could be more than `atLeast `?

Reply:
At this point in the code, no. See line 34.

Reply:
changed